vello/src/scene.rs

```diff
@@ -66,6 +66,9 @@ impl Scene {
     /// until the layer is popped.
     ///
     /// **However, the transforms are *not* saved or modified by the layer stack.**
+    ///
+    /// Clip layers (`blend` = [`Mix::Clip`]) should have an alpha value of 1.0.
+    /// For an opacity group with non-unity alpha, specify [`Mix::Normal`].
```

Comment:
Can we add a warning for this being violated?

Reply:
I believe clip is just an optimization so what about removing it from the public API and having normal+over+opaque encode as a clip?

Reply:
I'm open to adding a warning but don't know what the technology for that is.

And, as discussed in the [Zulip thread](https://xi.zulipchat.com/#narrow/stream/197075-gpu/topic/Vello.3A.20.60push_layer.60.20causing.20black.20rectangle/near/460112832), it's not just an optimization, it has different semantics when doing nontrivial blending. I'll improve the docs a bit. Also, as discussed in that thread, it's likely that support for non-isolated blend groups should become a feature request, but that's out of scope for this PR.